### PR TITLE
chore(ui): updated cli packages for TW (FEV-560)

### PR DESCRIPTION
**Pull request by semarche-kaltura**
This change is [Reviewable](https://reviewable.io/reviews/kaltura/playkit-js-transcript/56)

### CODE REVIEW

File: test/config.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "data": {
+    "partnerId": "",
+    "uiConfId": "",
+    "ks": "",
+    "widgetId": "",
+    "userId": "",
+    "entryId": "",
+    "switchMediaEntryId": ""
+  }
```
Comment (eitanavgil) [inline test/config.json:10]: remove

File: test/env.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "env": {
+    "production": {
+      "bundler": "https://cdnapisec.kaltura.com",
+      "serviceUrl": "https://cdnapisec.kaltura.com/api_v3",
+      "cdnUrl": "https://cfvod.kaltura.com"
```
Comment (eitanavgil) [inline test/env.json:6]: remove

Comment (semarche-kaltura) [reply test/env.json]: done

File: test/config.json (hunk 1 above)
Comment (semarche-kaltura) [reply test/config.json]: done

**Pull request merged**